### DISCUSSION TITLE
Integrating jinx with embark

### DISCUSSION THREAD

**Discussion by tillheilmann** [minad/jinx#213] [Q&A] [answered]
I tried to integrate jinx with embark with `(define-key embark-identifier-map "$" 'jinx-correct)`.

However, replacing the function `ispell-word` with `jinx-correct` in the keymap like this did not work for me. Calling it with `embark-act` simply does nothing on my machine. Calling it with `M-x` works like expected. My Emacs-fu is weak. Can anyone point me gently in the right direction for this? Thanks!

**Comment by minad** [comment, marked as answer]
I have this code in my config, but don't use it much. I hope this helps:

~~~ elisp
    (embark-define-overlay-target jinx category (eq %p 'jinx-overlay))
    (add-to-list 'embark-target-finders 'embark-target-jinx-at-point)
    (add-to-list 'embark-keymap-alist '(jinx jinx-repeat-map embark-general-map))
    (add-to-list 'embark-repeat-actions #'jinx-next)
    (add-to-list 'embark-repeat-actions #'jinx-previous)
    (add-to-list 'embark-target-injection-hooks (list #'jinx-correct #'embark--ignore-target))
~~~

**Reply by tillheilmann** [reply]
I do not understand how exactly this code works, but it does pretty much what I asked for, so thank you very much! (And thanks, btw, for jinx in general.)
